packages/services/f-braze-adapter/src/services/_tests/BrazeDispatcher.test.js

```diff
@@ -434,23 +332,23 @@ describe('BrazeDispatcher operation', () => {
                 expect(appboy.display.showInAppMessage).toHaveBeenCalledWith(message);
             });
 
-            describe('when showInAppMessage throws an error', () => {
-                const error = new Error('We have a problem');
-
-                beforeEach(() => {
-                    interceptInAppMessages.mockImplementation(() => {
-                        throw error;
-                    });
-                });
-
-                it('should log the error to the provided callback', async () => {
-                    // Act
-                    await interceptInAppMessagesHandler(message);
-
-                    // Assert
-                    expect(loggingCallback).toHaveBeenCalledWith(LOG_ERROR, `Error handling message - ${error}`, { message, error });
-                });
-            });
+            // describe('when showInAppMessage throws an error', () => {
```

Comment:
[`xdescribe()`/`describe.skip()`](https://jestjs.io/docs/api#describeskipname-fn) is your friend when temporarily disabling tests 😄